common/src/main/templates/io/netty/util/collection/KObjectHashMap.template

```diff
@@ -350,9 +350,10 @@ private int indexOf(int key) {
     /**
      * Returns the hashed index for the given key.
      */
-    private int hashIndex(int key) {
+    private int hashIndex(@k@ key) {
         // Allowing for negative keys by adding the length after the first mod operation.
-        return (key % keys.length + keys.length) % keys.length;
+        int hashCode = @HASH_CODE@;
```

Comment:
Can't remember if I ask on this line before...but can we inline this?  (no need for another variable).